docs/administration/vulnerability-scanning/_index.md

```diff
@@ -3,30 +3,29 @@ title: Vulnerability Scanning
 weight: 35
 ---
 
-Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://github.com/coreos/clair). To be able to use Trivy, Clair or both you must have enabled Trivy, Clair or both when you installed your Harbor instance (by appending installation options `--with-trivy`, `--with-clair` or both).
+Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://github.com/coreos/clair). 
+
+To use Trivy or Clair or both, you must enable Trivy, Clair, or both when you install your Harbor instance (by appending installation options `--with-trivy`, `--with-clair`, or both).
```

Comment:
has to be done in installation? is there any way to add a new scanner after harbor is deployed? @steven-zou

Reply:
I mean through the interrogation services

Reply:
See the section shown below. We talked about user can use their own Trivy and Clair.

Reply:
no edits needed here then